src/components/Swap/Summary/Details.tsx

```diff
@@ -86,14 +85,15 @@ export default function Details({ trade, slippage, impact }: DetailsProps) {
     return rows
   }, [
     feeOptions,
+    gasUseEstimateUSD,
     i18n.locale,
     impact,
-    inputCurrency,
+    inputCurrency.symbol,
     integrator,
-    lpFeeAmount,
     outputAmount,
     outputCurrency,
-    slippage,
+    slippage.allowed,
```

Comment:
is this actually desirable to have two comparisons vs one? not to be pedantic but i had to read a bit to figure out these changes.

Reply:
This is done through linting, so it is desirable in that sense.

If you can guarantee that a change to `slippage` will always occur if `slippage.allowed || slippage.warning` changes, then it is fine to just depend on `slippage`, but nothing in code (aside from our own conventions) actually guarantees this, so depending on them separately is technically correct and safer.